src/careamics/prediction_utils/prediction_outputs.py

```diff
@@ -41,15 +41,71 @@ def convert_outputs(predictions: list[Any], tiled: bool) -> list[NDArray]:
     return predictions_output
 
 
+def convert_outputs_pn2v(
+    predictions: list[Any], tiled: bool
+) -> tuple[list[NDArray], list[NDArray]]:
+    """
+    Convert the Lightning trainer outputs to the desired form.
+
+    This method allows stitching back together tiled predictions.
+
+    Parameters
+    ----------
+    predictions : list
+        Predictions that are output from `Trainer.predict`. Length of list the total
+        number of tiles divided by the batch size. Each element consists of a tuple of
+        ((prediction, mse), tile_info_list). 1st dimension of each tensor is the bs.
+        Length of tile info list is the batch size.
+
+    tiled : bool
+        Whether the predictions are tiled.
+
+    Returns
+    -------
+    tuple[list[NDArray], list[NDArray]]
+        Tuple of (predictions, mmse) where each is a list of arrays with axes SC(Z)YX.
+    """
+    if len(predictions) == 0:
+        return [], []
+
+    if tiled:
+        # Separate predictions and mmse, keeping tile info for each
+        pred_with_tiles = [
+            (pred, tile_info_list) for (pred, _), tile_info_list in predictions
+        ]
+        mse_with_tiles = [
+            (mse, tile_info_list) for (_, mse), tile_info_list in predictions
+        ]
```

Comment:
I would prefer the more straightforward for loop, for readability pruposes

```suggestion
pred_with_tiles=[]
mse_with_tiles = []
for (pred, mse), tile_info_list in predictions:
    pred_with_tiles.append((pred, tile_info_list))
    mse_with_tiles.append((mse, tile_info_list))
```


Alternatively, if you really love list iterations that make people read the code four times, this may work:

```
pred_with_tiles, mse_with_tiles = zip(
    *[((pred, tile_info_list), (mse, tile_info_list)) for (pred, mse), tile_info_list in predictions]
)
pred_with_tiles = list(pred_with_tiles)
mse_with_tiles = list(mse_with_tiles)
```

Reply:
Actually I can see the problem with the indentation here, probably the problem was on my end (my code suggestion did not have the indentation).

But you reverted to the previous code snippet?